### PR TITLE
Fix in-place edits to files being installed

### PR DESCRIPTION
**Pull request by LostDragonist**
Calling Pretty with GetDataFiles() and GetPlugins() was causing the
files to be extracted again and most of the work done by the script
to be thrown away.

### CODE REVIEW

**Comment by AnyOldName3** [comment]
There was some architectural reason why it had to work that way, although I don't remember it. I'll have a quick look into what it was, but I think the way this might be *supposed* to work would be to have the edit end up in the `ScriptReturnData`.

**Comment by AnyOldName3** [comment]
I've had a look at the vaguely-similar C#-only function `GenerateNewDataFile` and that seems to work similarly, i.e. the new file is created within the temporary directory the script acts on. The only mod that's known to use it also generates a BSA during install, which isn't supported, so the problem wasn't noticed there, either.

**Comment by AnyOldName3** [comment]
Looks like https://github.com/erri120/OMODFramework/issues/27 was the relevant issue. I don't really understand why it got opened, though - the overloads that let you pass an existing plugin and data directory existed all along during the 2.x era. I guess maybe I was tired and missed that there were separate overloads instead of a single implementation with default arguments.

Now I know that I at least thought this was impossible, and that's why it wasn't like that all along, I'm satisfied that there won't be any major fires from merging this. Yay!

File: src/installerOmod.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ QString InstallerOMOD::description() const
 
 MOBase::VersionInfo InstallerOMOD::version() const
 {
-  return MOBase::VersionInfo(1, 0, 0, MOBase::VersionInfo::RELEASE_PREALPHA);
+  return MOBase::VersionInfo(1, 1, 0, 0);
```
Comment (AnyOldName3) [inline src/installerOmod.cpp]: I'm pretty sure the last field here should be one of the `MOBase::VersionInfo::RELEASE_XXXXX` enum values.

Comment (LostDragonist) [reply src/installerOmod.cpp]: VersionInfo has several variants.  This is switching from
`VersionInfo(int major, int minor, int subminor, ReleaseType releaseType = RELEASE_FINAL);`
to
`VersionInfo(int major, int minor, int subminor, int subsubminor, ReleaseType releaseType = RELEASE_FINAL);`.  Doesn't really matter.

**Pull request merged**